### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by pokey** [tree-sitter/tree-sitter#2027] [Q&A] [answered]
Is there an official Discord for tree-sitter?  Might be nice to have a place to chat about all things tree-sitter related.  Fwiw @damieng just created ~https://discord.gg/KrRa5ATb~ (https://github.com/tree-sitter/tree-sitter-c-sharp/issues/273#issuecomment-1380411510), but I just wanted to see if there was already an official one that we had missed

**Comment by ahlinc** [comment, marked as answer]
Unexpirable invite link: https://discord.gg/w7nTvsVJhm

**Reply by bicepjai** [reply]
This seems not work, can you confirm is this discord is still alive ?

**Reply by sogaiu** [reply]
AFAICT, the server is still alive and though I don't know if it's a good test, I was able to access via the invite link.
